frontend/public/components/monitoring/receiver-forms/email-receiver-form.tsx

```diff
@@ -0,0 +1,297 @@
+              <div className="form-group">
+                <label className="control-label co-required">SMTP Smarthost</label>
+                <input
+                  className="pf-c-form-control"
+                  type="text"
+                  aria-describedby="email-smarthost-help"
+                  name="emailSmarthost"
+                  data-test-id="email-smarthost"
+                  value={formValues.emailSmarthost}
+                  onChange={(e) =>
+                    dispatchFormChange({
+                      type: 'setFormValues',
+                      payload: { emailSmarthost: e.target.value },
+                    })
+                  }
+                />
+                <div className="help-block" id="email-smarthost-help">
+                  Smarthost used for sending emails, including port number
+                </div>
+              </div>
+            </div>
+            <div className="col-sm-6">
+              <div className="form-group">
+                <label className="control-label co-required">SMTP Hello</label>
+                <input
+                  className="pf-c-form-control"
+                  type="text"
+                  aria-describedby="email-hello-help"
+                  name="emailHello"
+                  data-test-id="email-hello"
+                  value={formValues.emailHello}
+                  onChange={(e) =>
+                    dispatchFormChange({
+                      type: 'setFormValues',
+                      payload: { emailHello: e.target.value },
+                    })
+                  }
+                />
+                <div className="help-block" id="email-hello-help">
+                  The hostname to identify to the SMTP server
+                </div>
+              </div>
+            </div>
+          </div>
+          <div className="row">
+            <div className="col-sm-6">
+              <div className="form-group">
+                <label className="control-label">Auth Username</label>
+                <input
+                  className="pf-c-form-control"
+                  type="text"
+                  aria-describedby="email-auth-username-help"
+                  name="emailAuthUsername"
+                  data-test-id="email-auth-username"
+                  value={formValues.emailAuthUsername}
+                  onChange={(e) =>
+                    dispatchFormChange({
+                      type: 'setFormValues',
+                      payload: { emailAuthUsername: e.target.value },
+                    })
+                  }
+                />
+              </div>
+            </div>
+            <div className="col-sm-6">
+              <div className="form-group">
+                <label className="control-label">Auth Password (Using LOGIN and PLAIN)</label>
+                <input
+                  className="pf-c-form-control"
+                  type="password"
+                  aria-describedby="email-auth-password-help"
```

Comment:
I don't see an element with this ID

Reply:
There isn't a `help-block` for this/these fields, do I not need `aria-describedby` then?

Reply:
> There isn't a help-block for this/these fields, do I not need aria-describedby then?

Right. If there's nothing describing it, don't use `aria-describedby`. It's basically a broken link.

Reply:
fixed